reana_client/cli/files.py

```diff
@@ -122,8 +122,8 @@ def get_files(ctx, workflow, _filter,
 
 @click.command(
     'download',
-    help='Download all output files declared in the reana.yaml'
-         'specification or download files listed as '
+    help='Download workflow output files. Files are declared in the '
+         'reana.yaml specification or download files listed as '
```

Comment:
We cannot use `closes issue` in the commit message until we add usage examples for commands, see the second part of the issue.

Reply:
I am still working on it, that's why I moved it to 'In Work' column after I've made pull request.